zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/SparkConn.scala

```diff
@@ -0,0 +1,54 @@
+package edu.uci.ics.cloudberry.zion.model.impl
+import edu.uci.ics.cloudberry.zion.model.datastore.IDataConn
+import org.apache.spark.sql.SparkSession
+import org.apache.spark.sql.types._
+import play.api.libs.json._
+import play.api.libs.ws.{WSClient, WSResponse}
+import edu.uci.ics.cloudberry.util.Logging
+import org.apache.spark.sql.Encoders
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ExecutionContext, Future}
+/**
+  * Created by sicongliu on 17/5/19.
+  */
+
+class SparkConn(url: String)(implicit ec: ExecutionContext) extends IDataConn{
+  val spark = SparkSession.builder
+    .master(url)
+    .appName("Cloudberry")
+    .getOrCreate()
+
+  val schema = (new StructType).add("create_at", TimestampType).add("id",LongType).add("favorite_count", DoubleType).add("retweet_count ", DoubleType).add("in_reply_to_status", DoubleType).add("in_reply_to_user", DoubleType).add("text", StringType).add("lang", StringType).add("is_retweet", BooleanType).add("user_mentions",ArrayType(DoubleType)).add("hashtags",ArrayType(StringType)).add("geo_tag",(new StructType).add("stateID",IntegerType).add("countyID",IntegerType).add("cityID",IntegerType)).add("user",(new StructType).add("id",IntegerType).add("statues_count",DoubleType))
```

Comment:
since it is a fake one, can we add some comments about it and write the TODO to remove it in future?